### PR TITLE
ansible-test: remove alpine 3.17

### LINKED ISSUE #80418: Remove Alpine 3.17 from ansible-test
### Summary

Remove Alpine 3.17 from ansible-test after a 2-week transition period following the addition of Alpine 3.18 to ansible-test. This is a remote VM removal.

### Issue Type

Feature Idea

### Component Name

`ansible-test`

### PR DESCRIPTION
**Pull request by bcoca**
fixes #80418 

##### ISSUE TYPE

- Test Pull Request

##### COMPONENT NAME

ansible-test

### CODE REVIEW

**Comment by mattclay** [comment]
This should be announced before being merged.

For reference, here's the recent announcement for the Fedora change: https://github.com/ansible-collections/news-for-maintainers/issues/50

The tool to generate the announcement checklist has already been updated: https://github.com/ansible/ansible-azp-tools/pull/33

File: changelogs/fragments/a_test_rmv_alpine_317.yml
Comment (mattclay) [inline changelogs/fragments/a_test_rmv_alpine_317.yml]: We don't include CI changes in the changelog, since they don't make it into the sdist, just the ansible-test changes:

```suggestion
  - ansible-test - remove Alpine 3.17 from remotes
```

**Review by mattclay — CHANGES_REQUESTED** [review]

**Review by mattclay — APPROVED** [review]

**Comment by mattclay** [comment]
I converted this to a draft to help avoid accidental merges.

**Comment by bcoca** [comment]
https://github.com/ansible-collections/news-for-maintainers/issues/52

**Pull request merged**
